component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java

```diff
@@ -452,4 +462,35 @@ public static String getBaseURLSocialUserRest() {
     }
     return baseURLSocialUserRest;
   }
+
+  /**
+   * Gets platform language of current user. In case of any errors return null.
+   *
+   * @return the platform language
+   */
+  public static String getCurrentUserLanguage(String userId) {
+    try {
+      LocaleContextInfo localeCtx = LocaleContextInfoUtils.buildLocaleContextInfo(userId);
+      LocalePolicy localePolicy = ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(LocalePolicy.class);
+      String lang = null;
+      if(localePolicy != null) {
+        Locale locale = localePolicy.determineLocale(localeCtx);
+        lang = locale.toString();
+      }
+      return lang;
+    } catch (Exception e) {
+      LOG.error("Error searching user " + userId, e);
+      return null;
+    }
+  }
+
+  /**
+   * Gets the resource bundle label.
+   *
+   * @return the label
+   */
+  public static String getResourceBundleLabel(Locale locale, String label) {
```

Comment:
I think that there is an Utils method added for that ?

Reply:
It can't be used in core component !